Makefile.am

```diff
@@ -813,6 +813,20 @@ endif
 
 check: $(CHECK_DEPS)
 
+install-all: install-kernel |  install-image |  install
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	curl https://raw.githubusercontent.com/clearcontainers/tests/master/.ci/install_clear_image.sh \
```

Comment:
I think reducing the necessity to run as `root` to a minimum makes sense as a general plan.

Running a dynamically downloaded script as root will raise a security flag, so we should probably address it. I will note that I think the 'level of trust' of that shell script is currently probably the same as that of the `Makefile` that is downloading and running it - they both come from github repos managed by the same group - but, reducing the attack surface is always a good thing.